spec/features/finance/npq/course_payment_breakdown_spec.rb

```diff
@@ -182,11 +204,11 @@ def then_i_should_see_correct_breakdown_summary(npq_lead_provider, npq_contract)
       expect(page).to have_content("Recruitment target")
       expect(page).to have_content(npq_contract.recruitment_target)
       expect(page).to have_content("Current participants")
-      expect(page).to have_content(ParticipantDeclaration::NPQ.neither_paid_nor_voided_lead_provider_and_course(npq_lead_provider, npq_contract.course_identifier).count)
+      expect(page).to have_content(ParticipantDeclaration::NPQ.where(statement: statement, cpd_lead_provider: cpd_lead_provider, course_identifier: npq_contract.course_identifier, state: "payable").count)
       expect(page).to have_content("Total paid")
-      expect(page).to have_content(expected_total_paid(npq_contract))
+      expect(page).to have_content(ParticipantDeclaration::NPQ.where(statement: statement, cpd_lead_provider: cpd_lead_provider, course_identifier: npq_contract.course_identifier, state: "payable").count)
       expect(page).to have_content("Total not paid")
-      expect(page).to have_content(ParticipantDeclaration::NPQ.submitted_for_lead_provider_and_course(npq_lead_provider, npq_contract.course_identifier).count)
+      # TODO: to be implemented later
```

Comment:
@asmega what's to implement later? Would you be make this a bit more descriptive please?

Reply:
This is referring the previous line `Total not paid` which is not implemented as it is not currently possible